drift_dev/lib/src/model/sql_query.dart

```diff
@@ -157,6 +158,28 @@ abstract class SqlQuery {
 
     return resultClassName;
   }
+
+  /// Returns all found elements, from this query an all nested queries. The
+  /// elements returned by this method are in no particular order, thus they
+  /// can only be used to determine the method parameters.
+  ///
+  /// This method makes some effort to remove duplicated parameters. But only
+  /// by comparing the dart name.
+  List<FoundElement> elementsWithNestedQueries() {
+    final elements = List.of(this.elements);
+
+    final subQueries = resultSet?.nestedResults.whereType<NestedResultQuery>();
```

Comment:
This (and the early bail-out in the collecting visitor when seeing nested result queries) wouldn't supported nested nested queries, like `LIST (SELECT ..., LIST(...))`. For our own sanity, I don't think we have to support that, but then we should add a linter flagging invalid usages of `LIST` somewhere.

Reply:
Already did something like this earlier today, just pushed it :)